tests/agent_server/test.py

```diff
@@ -63,43 +64,31 @@ async def get_resource_action(
         before = before or datetime.datetime.now()
         after = after or datetime.datetime.fromtimestamp(0)
 
+        def args_as_dict(**kwargs):
+            return kwargs
+
         while before > after:
+            kwargs = args_as_dict(
```

Comment:
I think this also works

```suggestion
            kwargs = dict(
```

Reply:
Done in https://github.com/inmanta/inmanta-core/pull/2762/commits/a44783eeef9cb2b495df81a40cbcfa47b5ad31a2